app/models/bike.rb

```diff
@@ -752,4 +756,12 @@ def cycle_type_name
   def propulsion_type_name
     PropulsionType.new(propulsion_type).name
   end
+
+  def geo_data
+    stolen_record_location = find_current_stolen_record&.address(override_show_address: true)
```

Comment:
I would like the current stolen record location to override the location info, if it's set and the location info isn't

Reply:
Updated in deb6c69c